### ISSUE TITLE
About the accuracy of the released model

### ISSUE THREAD

**Issue by Cardinal2376** [mks0601/3DMPPE_POSENET_RELEASE#61]
Hi, I have download the released model "snapshot_24.pth" for Human36 protocol2 and tested it with your test.py code. But I find that the error is far larger than the number you reported. Do I need to change any parameters for Human36 in the config.py? Or is there any bugs for my evaluation?
I have checked the protocol setting in Human36M.py is 2.
![Screenshot from 2020-08-03 20-23-31](https://user-images.githubusercontent.com/19239434/89181969-4c16eb00-d5c7-11ea-835c-f82b3c4e8b8e.png)

**Comment by mks0601** [comment]
![스크린샷 2020-08-03 오후 10 01 16](https://user-images.githubusercontent.com/13601723/89185226-dca7f800-d5d4-11ea-9deb-0004f29d8e01.png)

This is what I've got just before.
1) clone the repo
2) download the pre-trained model
3) put the model to output/model_dump
4) go to main
5) python test.py --gpu 0-1 --test_epoch 24

**Comment by Cardinal2376** [comment]
After hardworking, I finally find out one reason: the torchvision version. Previously I used torchvision 0.2.0 and the model output did not align with the image, and when it is upgrade to 0.4.0, it looks well.
But still, I do not get the accuracy like you. Could you please describe all the environments in detail?
![0_output](https://user-images.githubusercontent.com/19239434/89271818-d31f9e00-d66f-11ea-9d65-24ee582a90b2.jpg)
![0_output](https://user-images.githubusercontent.com/19239434/89271829-d6b32500-d66f-11ea-91d9-64b05b2c0df5.jpg)

**Comment by mks0601** [comment]
there are many packages installed in my machines, so I cannot tell you all the things :( I'm using python 3.7 and pytorch 1.3.1

**Comment by Cardinal2376** [comment]
Ok, I understand. Here is my environment setting. Is there any different between yours?
Thank you for your quick reply!

Package         Version
--------------- -------------------
certifi         2020.6.20
chardet         3.0.4
cPython         0.0.6
cycler          0.10.0
future          0.18.2
idna            2.10
kiwisolver      1.2.0
matplotlib      3.3.0
numpy           1.19.1
opencv-python   4.3.0.36
Pillow          7.2.0
pip             20.2
pycocotools     2.0
pymongo         3.11.0
pyparsing       2.4.7
python-dateutil 2.8.1
requests        2.24.0
setuptools      49.2.1.post20200802
six             1.15.0
torch           1.3.1
torchvision     0.4.2
tqdm            4.48.2
urllib3         1.25.10
wheel           0.34.2

**Comment by Cardinal2376** [comment]
I find out the problem. I did not update he annotations for Human36. The accuracy is matched as yours, thank you very much!
![Screenshot from 2020-08-04 17-56-59](https://user-images.githubusercontent.com/19239434/89281119-2ef02400-d67c-11ea-8375-8dfefbae0c6d.png)
